documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,38 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our release notes, and this directory contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Each file should be named like ``<ISSUE_OR_PR>.<TYPE>.rst``,
+where ``<ISSUE_OR_PR>`` is an issue or a pull request number -
+whatever is most useful to link to,
+and ``<TYPE>`` is one of:
+
+* ``feature``: New user-facing feature.
+* ``bugfix``: A bug fix.
+* ``doc``: Documentation improvement.
+* ``removal``: Deprecation or removal of public API or behavior.
```

Comment:
I think we might want a category for behavior changes, such as #1375 which isn't really a feature, but is a user-visible change that might be worth at least mentioning

Reply:
I just added that locally and will push once my network problems are over.

Reply:
``change`` is now in.